stl/inc/type_traits

```diff
@@ -1473,6 +1473,14 @@ _NODISCARD constexpr conditional_t<!is_nothrow_move_constructible_v<_Ty> && is_c
     return _STD move(_Arg);
 }
 
+// std::invoke isn't constexpr in C++17, and implementers are forbidden from "strengthening"
+// constexpr, yet both std::apply and std::visit are required to be constexpr and have invoke-like behavior. We solve

```

Comment:
WG21-N4842 [constexpr.functions]/1 "An implementation shall not declare any standard library function signature as `constexpr` except for those where it is explicitly required."